app/presenters/tree_builder_utilization.rb

```diff
@@ -55,7 +55,7 @@ def x_get_tree_vandt_datacenter_kids(object)
         objects += rbac_filtered_sorted_objects(f.vms_and_templates, "name")
       elsif f.name == "host"            # Don't count host folder children
       else                              # add in other folders
-        objects += Rbac.filtered([f], :match_via_descendants => VmOrTemplate)
+        objects << Rbac.filter(f, :match_via_descendants => VmOrTemplate)
```

Comment:
I'm confused.  This one wasn't grabbing just the first result.  Was it?  Is this a change in behavior?